File: 01/csharp/exercise14.cs
class Operations {
    public static double squared(double num) {
        return Math.Pow(num, 2);
    }
    
    public static double cubed(double num) {
        return Math.Pow(num, 3);    
    }
}

class MainClass {
    public static void Main() {
        Console.WriteLine("Escriba un número: ");
        double input =  Convert.ToDouble(Console.ReadLine());
        double squaredNum = Operations.squared(input);
        double cubedNum = Operations.cubed(input);

        Console.WriteLine($"{input}^2 = {squaredNum}");
        Console.WriteLine($"{input}^3 = {cubedNum}");
    }
}

File: 01/python/exercise14.py
class Operations:
    @staticmethod
    def squared(num):
        return num**2

    @staticmethod
    def cubed(num):
        return num**3

input = int(input('Escriba un número: '))
squaredNum = Operations.squared(input)
cubedNum = Operations.cubed(input)

print(f'{input}^2 = {squaredNum}')
print(f'{input}^3 = {cubedNum}')

